code/modules/xenomorph/xeno_structures.dm

```diff
@@ -144,86 +153,23 @@
 	associated_hive.xeno_message("<span class='xenoannounce'>Our [name] has detected a nearby hostile [hostile] at [get_area(hostile)] (X: [hostile.x], Y: [hostile.y]). [name] has [obj_integrity]/[max_integrity] Health remaining.</span>", 2, FALSE, hostile, 'sound/voice/alien_help1.ogg')
 	COOLDOWN_START(src, silo_proxy_alert_cooldown, XENO_HEALTH_ALERT_COOLDOWN) //set the cooldown.
 
+///Signal handler to tell the silo it can start maturing
+/obj/structure/resin/silo/proc/start_maturing()
+	SIGNAL_HANDLER
+	does_mature = TRUE
+	if(!CHECK_BITFIELD(datum_flags, DF_ISPROCESSING))
+		START_PROCESSING(SSslowprocess, src)
```

Comment:
wont this only make silos pre shutter work ? what about silos made after it ?

Reply:
You are right, mistake. It should be a static or something